pallets/ocex/src/lib.rs

```diff
@@ -408,6 +411,18 @@ pub mod pallet {
 				Error::<T>::AmountOverflow
 			);
 
+			//enclave will only support min volume of 10^-8
+			//if trading pairs volume falls below it will pass a UnderFlow Error
+			ensure!(
+				min_order_price.saturated_into::<u128>() > MIN_CONFIG &&
+					min_order_qty.saturated_into::<u128>() > MIN_CONFIG &&
+					min_order_price
+						.saturated_into::<u128>()
+						.saturating_mul(min_order_qty.saturated_into::<u128>()) >
+						MIN_CONFIG,
+				Error::<T>::TradingPairConfigUnderflow
```

Comment:
Yes you are right, if min_price and min_qty are checked the min_volume check would work by default. Should i remove it @Gauthamastro.

Reply:
We are not asserting a min value for min_price and min_qty, as it can be 1  ( u128 ) and MIN_CONFIG for min_qty. Then, that check will still pass, but our downstream libraries will break. We have to ensure at least as a sanity check that min_price, min_qty and min_volume doesn't fall under MIN_CONFIG.

Reply:
Ok, so @serhii-temchenko we will keep it.